core/arch/arm/plat-nuvoton/npcm845x_utils.h

```diff
@@ -0,0 +1,37 @@
+/*
+ *  Copyright (c) 2022 Nuvoton Technology Corp.
+ *
+ * See file CREDITS for list of people who contributed to this
+ * project.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 2 of
+ * the License, or (at your option) any later version.
```

Comment:
OP-TEE OS will not merge GPL licensed code. Please reconsider you licensing model.